keras_cv/models/stable_diffusion/stable_diffusion.py

```diff
@@ -31,25 +28,22 @@
 from keras_cv.models.stable_diffusion.constants import _ALPHAS_CUMPROD
 from keras_cv.models.stable_diffusion.constants import _UNCONDITIONAL_TOKENS
 from keras_cv.models.stable_diffusion.decoder import Decoder
-from keras_cv.models.stable_diffusion.diffusion_model import DiffusionModel
+from keras_cv.models.stable_diffusion.diffusion_model import DiffusionModel, DiffusionModelV2
 from keras_cv.models.stable_diffusion.image_encoder import ImageEncoder
-from keras_cv.models.stable_diffusion.text_encoder import TextEncoder
+from keras_cv.models.stable_diffusion.text_encoder import TextEncoder, TextEncoderV2
 
 MAX_PROMPT_LENGTH = 77
 
 
 class StableDiffusion:
     """Keras implementation of Stable Diffusion.
-
     Note that the StableDiffusion API, as well as the APIs of the sub-components
```

Comment:
It looks like your auto-formatter has removed all of these blank lines in the docs -- would you mind re-adding them?